src/global_physics.py

```diff
@@ -0,0 +1,15 @@
+import math
+import numpy as np
+import src.common_constants as common_constants
+
+class GlobalPhysics():
+    def __init__(self):
+        self.gravity = common_constants.CommonConstants.GRAVITATION
+        self.air_density = common_constants.CommonConstants.AIR_DENSITY
+        self.objects_assected_by_physics = []
```

Comment:
```suggestion
        self.physical_objects = []
```
shorter, convenient for an object that will be used frequently